src/Requests.php

```diff
@@ -590,6 +590,8 @@ public static function request_multiple($requests, $options = []) {
 
 		unset($request);
 
+		$options['hooks']->dispatch('multiple.request.before_request_multiple', [&$requests]);
```

Comment:
`multiple.request.complete` gets every response one by one rather than all at once. Since this will be used to complement the `multiple.request.complete` hook it should IMHO also get the requests one by one.